.github/workflows/testBuild.yml

```diff
@@ -25,30 +25,29 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.OS_BOTIFY_TOKEN }}
           username: ${{ github.actor }}
           team: expensify-expensify
-      - name: Remove label if it was added by an unauthorized user
-        if: ${{ !fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && github.event.label.name == 'Ready To Build' }}
-        uses: actions-ecosystem/action-remove-labels@v1
-        with:
-          labels: 'Ready To Build'
-      - name: Throw exception if label was added by an unauthorized user
-        if: ${{ !fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && github.event.label.name == 'Ready To Build' }}
-        run: |
-          echo "The 'Ready To Build' label was added by an unauthorized user"
-          exit 1
+
+      - id: hasReadyToBuildLabel
+        name: Set HAS_READY_TO_BUILD_LABEL flag
+        run: echo "HAS_READY_TO_BUILD_LABEL=$(gh pr view "$PULL_REQUEST_NUMBER" --json labels --jq '.labels[].name' | grep -q 'Ready To Build' && echo 'true')" >> "$GITHUB_OUTPUT"
+        env:
+          PULL_REQUEST_NUMBER: ${{ github.event.number || github.event.inputs.PULL_REQUEST_NUMBER }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
       - id: readyToBuild
         name: Set READY_TO_BUILD flag
-        run: echo "READY_TO_BUILD=${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) || contains(github.event.pull_request.labels.*.name, 'Ready To Build') }}" >> "$GITHUB_OUTPUT"
+        run: echo "READY_TO_BUILD=${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && steps.hasReadyToBuildLabel.outputs.HAS_READY_TO_BUILD_LABEL == 'true' }}" >> "$GITHUB_OUTPUT"
```

Comment:
```suggestion
        run: echo "READY_TO_BUILD=${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && fromJSON(steps.hasReadyToBuildLabel.outputs.HAS_READY_TO_BUILD_LABEL) }}" >> "$GITHUB_OUTPUT"
```

Reply:
Actually, I would just get rid of this step entirely and evaluate this condition at the job level:

```yaml
jobs:
  validateActor:
    outputs:
      READY_TO_BUILD: ${{ fromJSON(steps.isUserTeamMember.outputs.isTeamMember) && fromJSON(steps.hasReadyToBuildLabel.outputs.HAS_READY_TO_BUILD_LABEL) }}
```